### ISSUE TITLE
nnf.pad error

### ISSUE THREAD

**Issue by wangkaiwan** [junyuchen245/TransMorph_Transformer_for_Medical_Image_Registration#6]
Hi Junyu,

Thanks for sharing your interesting work! It does help a lot for my project. However, I found some tiny errors related to [nnf.pad ](https://pytorch.org/docs/stable/generated/torch.nn.functional.pad.html) function in your [code](https://github.com/junyuchen245/TransMorph_Transformer_for_Medical_Image_Registration/blob/main/TransMorph/models/TransMorph.py). So I made the below modifications:
```python
x = nnf.pad(x, (0, 0, pad_f, pad_h, pad_t, pad_b, pad_l, pad_r)) # Line 219
# x = nnf.pad(x, (0, 0, pad_l, pad_r, pad_t, pad_b, pad_f, pad_h))
        
if pad_input: #Line 285
    x = nnf.pad(x, (0, 0, 0, T % 2, 0, W % 2, 0, H % 2))
    # x = nnf.pad(x, (0, 0, 0, W % 2, 0, H % 2, 0, T % 2))

_, _, H, W, T = x.size() # Line 439
if T % self.patch_size[2] != 0:
    x = nnf.pad(x, (0, self.patch_size[2] - T % self.patch_size[2]))
if W % self.patch_size[1] != 0:
    x = nnf.pad(x, (0, 0, 0, self.patch_size[1] - W % self.patch_size[1]))
if H % self.patch_size[0] != 0:
    x = nnf.pad(x, (0, 0, 0, 0, 0, self.patch_size[0] - H % self.patch_size[0]))
# if W % self.patch_size[1] != 0:
#     x = nnf.pad(x, (0, self.patch_size[1] - W % self.patch_size[1]))
# if H % self.patch_size[0] != 0:
#     x = nnf.pad(x, (0, 0, 0, self.patch_size[0] - H % self.patch_size[0]))
# if T % self.patch_size[0] != 0:
#     x = nnf.pad(x, (0, 0, 0, 0, 0, self.patch_size[0] - T % self.patch_size[0]))
```
Best,
Woaka

**Comment by junyuchen245** [comment]
Hi @wangkaiwan,

I apologize for the bugs and appreciate your notification.

I modified the code and it seems like the bugs had no effect on the pre-trained models' accuracy.

Thanks,
Junyu

**Comment by wangkaiwan** [comment]
@junyuchen245, yes, the pre-trained models worked well, and I found these tiny bugs when I modified the model structure.

Thanks again for the excellent work!

**Comment by junyuchen245** [comment]
Thanks!
